contracts/Liquidator/BUSDLiquidator.sol

```diff
@@ -0,0 +1,188 @@
+    /// @param token The address of the token to recover
+    /// @custom:access Only Governance
+    function sweepToken(IERC20Upgradeable token) external onlyOwner {
+        token.safeTransfer(msg.sender, token.balanceOf(address(this)));
+    }
+
+    /// @dev Unpauses the liquidation on the BUSD market, liquidates the borrower's debt,
+    /// and pauses the liquidations back
+    /// @param borrower The borrower whose debt should be liquidated
+    /// @param repayAmount The amount to repay
+    /// @param vTokenCollateral The collateral to seize from the borrower
+    function _unpauseAndLiquidate(address borrower, uint256 repayAmount, IVToken vTokenCollateral) internal {
+        address[] memory vTokens = new address[](1);
+        vTokens[0] = address(vBUSD);
+        IComptroller.Action[] memory actions = new IComptroller.Action[](1);
+        actions[0] = IComptroller.Action.LIQUIDATE;
+
+        comptroller._setActionsPaused(vTokens, actions, false);
+        _liquidateBorrow(borrower, repayAmount, vTokenCollateral);
+        comptroller._setActionsPaused(vTokens, actions, true);
+    }
+
+    /// @dev Performs the actual liquidation, transferring BUSD from the sender to this contract,
+    /// repaying the debt, and transferring the seized collateral to the sender and the treasury
+    /// @param borrower The borrower whose debt should be liquidated
+    /// @param repayAmount The amount to repay
+    /// @param vTokenCollateral The collateral to seize from the borrower
+    function _liquidateBorrow(address borrower, uint256 repayAmount, IVToken vTokenCollateral) internal {
+        ILiquidator liquidatorContract = ILiquidator(comptroller.liquidatorContract());
+        IERC20Upgradeable busd = IERC20Upgradeable(vBUSD.underlying());
+
+        uint256 actualRepayAmount = _transferIn(busd, msg.sender, repayAmount);
+        approveOrRevert(busd, address(liquidatorContract), actualRepayAmount);
+        uint256 balanceBefore = vTokenCollateral.balanceOf(address(this));
+        liquidatorContract.liquidateBorrow(address(vBUSD), borrower, actualRepayAmount, vTokenCollateral);
+        uint256 receivedAmount = vTokenCollateral.balanceOf(address(this)) - balanceBefore;
+        approveOrRevert(busd, address(liquidatorContract), 0);
+
+        (uint256 liquidatorAmount, uint256 treasuryAmount) = _computeShares(receivedAmount);
+        vTokenCollateral.safeTransfer(msg.sender, liquidatorAmount);
+        vTokenCollateral.safeTransfer(treasury, treasuryAmount);
+    }
+
+    /// @dev Transfers tokens to this contract and returns the actual transfer amount
+    /// @param token The token to transfer
+    /// @param from The account to transfer from
+    /// @param amount The amount to transfer
+    /// @return actualAmount The actual amount transferred
+    function _transferIn(
+        IERC20Upgradeable token,
+        address from,
+        uint256 amount
+    ) internal returns (uint256 actualAmount) {
```

Comment:
I think we don't need to name the returned variable, because we are using `return`

Reply:
Fixed in [a59de0b](https://github.com/VenusProtocol/venus-protocol/pull/362/commits/a59de0bcaa024ded6f728972d316ee5c880f0f4f)